scss/_variables.scss

```diff
@@ -448,7 +448,8 @@ $custom-file-box-shadow:    $input-box-shadow !default;
 $custom-file-button-color:  $custom-file-color !default;
 $custom-file-button-bg:     $input-group-addon-bg !default;
 
-// stylelint-disable
+// stylelint-disable function-parentheses-newline-inside
```

Comment:
Can you just combine the rules with a comma between?

Reply:
I did not know it. Everyday learn something new. Thank you.